wolox-books/src/app/components/auth/auth.component.html

```diff
@@ -0,0 +1,4 @@
+<nav class="container-nav">
+  <img class="logo-wolox" alt="logo-wolox" src="assets/LogoWolox-Original.png">
```

Comment:
If the logo is going to redirect to home, you should put the image inside an anchor tag